test/extensions/http/header_validators/envoy_default/http2_header_validator_test.cc

```diff
@@ -438,15 +438,6 @@ TEST_F(Http2HeaderValidatorTest, ValidateResponseHeaderMapEmptyGenericName) {
                              UhvResponseCodeDetail::get().EmptyHeaderName);
 }
 
-TEST_F(Http2HeaderValidatorTest, ValidateRequestTrailersAllowUnderscoreHeadersByDefault) {
```

Comment:
This test duplicates https://github.com/envoyproxy/envoy/blob/bf270142edd04d91d747864d85b4c0e731668b23/test/extensions/http/header_validators/envoy_default/http2_header_validator_test.cc#L729